flink-dist/src/main/assemblies/opt.xml

```diff
@@ -168,6 +168,14 @@
 			<destName>flink-python_${scala.binary.version}-${project.version}.jar</destName>
 			<fileMode>0644</fileMode>
 		</file>
+
+		<!-- ML (API & LIB)  -->
+		<file>
```

Comment:
I agree that there should be an uber module. The current packaging also appears to be incorrect since flink-ml-api is _not_ bundled in flink-ml-lib anyway.

Reply:
Hi @dianfu @zentol Thanks for your advice.
I would fine with adding an uber module. But I think there are some differences between the table module and the ml module. For table module, it needs a uber module to packages all table sub-modules plus other modules, e.g., cep. While for ml, there are no such requirements.

Reply:
Where exactly do they differ? They both have multiple sub-modules that you want to ship as a single jar.

Reply:
For the table module, it makes no sense to let any sub-modules to bundle cep, so a uber module is used to create the bundle jar. but for ml module, it makes sense to let flink-ml-lib to bundle the flink-ml-api module, so that we can simply put the flink-ml-lib jar into the opt. 

Anyway, I have created an uber module for the flink-ml in the latest pr.

Reply:
the table-uber module existed before it bundled cep though.

Reply:
ok, let's add the uber module.